trams_api_spec/trams_api_spec.yaml

```diff
@@ -95,7 +95,7 @@ paths:
       tags:
       - "Establishment"
       summary: "Get academy by UK Provider Reference Number(UKPRN)"
-      description: ""
+      #description: ""
```

Comment:
Could we fill in the description rather than comment it out? There's loads of empty description fields but it's only because I didn't know what it would be - I think we understand things well enough now to add them as we go

Reply:
I think the comment was just a typo 😅 but yeah, I'll add a description to the field

Reply:
I've removed the description entirely - there's a summary right above it, so it felt like it was just a repeated field